package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@microsoft/vscode-serial-monitor-api",
+  "version": "0.1.0",
+  "description": "Public API for vscode-serial-monitor",
+  "keywords": [
+    "vscode",
+    "serial",
+    "api",
+    "vscode-serial-monitor",
+    "monitor",
+    "serial monitor"
+  ],
+  "bugs": {
+    "url": "https://github.com/microsoft/vscode-serial-monitor/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/vscode-serial-monitor"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Microsoft Corporation"
+  },
+  "publisher": "ms-vscode",
+  "main": "dist/api.js",
+  "typings": "dist/api.d.ts",
+  "scripts": {
+    "build": "tsc -p ./",
```

Comment:
Right now I don't think we lint in CI. Consider either adding a lint step, or changing the build step to `npm run lint && tsc -p ./`.

Reply:
Changing.